docs/download_files.md

```diff
@@ -160,25 +160,23 @@ This includes a summary of the types of libraries (e.g., single-cell, single-nuc
 Every download also includes the individual [QC report](#qc-report) and, if applicable, [cell type annotation reports](#cell-type-report) for each library included in the merged object.
 
 ### Download folder structure for `SingleCellExperiment` merged downloads:
-_image pending_
+![project download folder](images/merged-project-download-folder.png)
 
 ### Download folder structure for `AnnData` merged downloads:
-_image pending_
+![project download folder](images/merged-anndata-project-download-folder.png)
 
 ### Download folder structure for `AnnData` merged downloads with CITE-seq (ADT) data:
-_image pending_
-
-
-If downloading a project containing bulk RNA-seq data, two tab-separated value files, `bulk_quant.tsv` and `bulk_metadata.tsv`, will be included in the project download.
-The `bulk_quant.tsv` file contains a gene by sample matrix (each row a gene, each column a sample) containing raw gene expression counts quantified by Salmon.
-The `bulk_metadata.tsv` file contains associated metadata for all samples with bulk RNA-seq data.
+![project download folder](images/merged-anndata-project-citeseq-download-folder.png)
 
 ### Merged object metadata
 
 Similar to downloading the project with individual files for each sample, downloading the project as a merged object includes a single `single_cell_metadata.tsv` file containing metadata for all samples included in the download.
 For a full description of this file's contents, refer to the [metadata section above ](#metadata).
 
-
+If downloading a project containing bulk RNA-seq data, two tab-separated value files, `bulk_quant.tsv` and `bulk_metadata.tsv`, will be included in the project download.
+The `bulk_quant.tsv` file contains a gene by sample matrix (each row a gene, each column a sample) containing raw gene expression counts quantified by Salmon.
```

Comment:
I just feel like its weird to talk about gene expression quantification in the metadata section. I almost think you could just remove this section and put all of the contents before any of the images. That's how it is in the main project download section.

Reply:
> I almost think you could just remove this section and put all of the contents before any of the images. That's how it is in the main project download section.

👍